src/Migrations/M036RemoveEnFormOptions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Remove the Theme Editor feature flag from Planet 4 features.
+ */
+class M036RemoveEnFormOptions extends MigrationScript
```

Comment:
Should we migrate the Counter blocks of type Progress Bar Inside EN Form, as well?

Reply:
I checked the EN Form blocks from the blocks report sheet and we don't have any active instance of embedded counter.

Reply:
Same with the NROs' websites?

Reply:
Yes, the blocks report is fetching that information from all websites